packages/utils/time.ts

```diff
@@ -3,8 +3,8 @@
 // added here, and should be based on dayjs (not moment)
 // -----------------------------------------------------------------------------------------------
 
-import dayjs = require('dayjs');
-import dayJsRelativeTime = require('dayjs/plugin/relativeTime');
+import * as dayjs from 'dayjs';
+import * as dayJsRelativeTime from 'dayjs/plugin/relativeTime';
```

Comment:
The `import = require` import style was failing [here](https://github.com/laurent22/joplin/actions/runs/9601789526/job/26481121502#step:8:156):
```
error ../utils/time.ts: /home/runner/work/joplin/joplin/packages/utils/time.ts: `import dayjs = require('dayjs')` is not supported by @babel/plugin-transform-typescript
Please consider using `import dayjs from 'dayjs';` alongside Typescript's --allowSyntheticDefaultImports option.
SyntaxError: /home/runner/work/joplin/joplin/packages/utils/time.ts: `import dayjs = require('dayjs')` is not supported by @babel/plugin-transform-typescript
Please consider using `import dayjs from 'dayjs';` alongside Typescript's --allowSyntheticDefaultImports option.
  4 | // -----------------------------------------------------------------------------------------------
  5 |
> 6 | import dayjs = require('dayjs');
  4 | // -----------------------------------------------------------------------------------------------
  5 |
> 6 | import dayjs = require('dayjs');
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  7 | import dayJsRelativeTime = require('dayjs/plugin/relativeTime');
  8 |
  9 | const supportedLocales: Record<string, unknown> = {.
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  7 | import dayJsRelativeTime = require('dayjs/plugin/relativeTime');
  8 |
  9 | const supportedLocales: Record<string, unknown> = {
    at File.buildCodeFrameError (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@react-native/metro-babel-transformer/node_modules/@babel/core/lib/transformation/file/file.js:205:12)
    at NodePath.buildCodeFrameError (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/path/index.js:105:21)
    at PluginPass.TSImportEqualsDeclaration (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/plugin-transform-typescript/lib/index.js:397:22)
    at newFn (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/visitors.js:153:14)
    at NodePath._call (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/path/context.js:46:20)
    at NodePath.call (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/path/context.js:36:17)
    at NodePath.visit (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/path/context.js:82:31)
    at TraversalContext.visitQueue (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/context.js:86:16)
    at TraversalContext.visitMultiple (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/context.js:61:17)
    at TraversalContext.visit (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/context.js:107:19)
    at traverseNode (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/traverse-node.js:18:17)
    at NodePath.visit (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/path/context.js:88:52)
    at TraversalContext.visitQueue (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/context.js:86:16)
    at TraversalContext.visitSingle (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/context.js:65:19)
    at TraversalContext.visit (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/context.js:109:19)
    at traverseNode (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/traverse-node.js:18:17)
    at traverse (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@babel/traverse/lib/index.js:49:34)
    at transformFile (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@react-native/metro-babel-transformer/node_modules/@babel/core/lib/transformation/index.js:81:29)
    at transformFile.next (<anonymous>)
    at run (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@react-native/metro-babel-transformer/node_modules/@babel/core/lib/transformation/index.js:24:12)
    at run.next (<anonymous>)
    at /home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@react-native/metro-babel-transformer/node_modules/@babel/core/lib/transform-ast.js:23:41
    at Generator.next (<anonymous>)
    at evaluateSync (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/gensync/index.js:251:28)
    at sync (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/gensync/index.js:89:14)
    at stopHiding - secret - don't use this - v1 (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@react-native/metro-babel-transformer/node_modules/@babel/core/lib/errors/rewrite-stack-trace.js:46:12)
    at transformFromAstSync (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@react-native/metro-babel-transformer/node_modules/@babel/core/lib/transform-ast.js:44:83)
    at Object.transform (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/@react-native/metro-babel-transformer/src/index.js:217:7)
    at transformJSWithBabel (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/metro-transform-worker/src/index.js:285:45)
    at Object.transform (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/metro-transform-worker/src/index.js:408:18)
    at transformFile (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/metro/src/DeltaBundler/Worker.flow.js:54:36)
    at Object.transform (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/metro/src/DeltaBundler/Worker.flow.js:30:10)
    at execFunction (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/jest-worker/build/workers/processChild.js:149:17)
    at execHelper (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/jest-worker/build/workers/processChild.js:137:5)
    at execMethod (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/jest-worker/build/workers/processChild.js:140:5)
    at process.messageListener (/home/runner/work/joplin/joplin/packages/app-mobile/node_modules/jest-worker/build/workers/processChild.js:44:7)
    at process.emit (node:events:517:28)
    at emit (node:internal/child_process:944:14)

> Task :app:createBundleReleaseJsAndAssets FAILED
```